docker-compose.yml

```diff
@@ -96,7 +96,7 @@ services:
       - ~/.sbt:/root/.sbt:cached
       - ~/.cache/coursier:/root/.cache/coursier:cached
     environment:
-      - SBT_OPTS=-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005 -Dfile.encoding=UTF-8 -Xms512m -Xmx1536m -Xss2m -XX:ReservedCodeCacheSize=256m -XX:+TieredCompilation -XX:+CMSClassUnloadingEnabled -XX:+UseConcMarkSweepGC
+      - SBT_OPTS=-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005 -Dfile.encoding=UTF-8 -Xms512m -Xmx1536m -Xss2m -XX:ReservedCodeCacheSize=512m -XX:+TieredCompilation -XX:+CMSClassUnloadingEnabled -XX:+UseConcMarkSweepGC
```

Comment:
The 512m here should also be reverted

Reply:
Okay resolving this convo. In discussion with @voropaevp it makes more sense to fix these SBT related problems in another PR. I didn't realize that @voropaevp is using a Windows system so it may be possible that docker is doing something odd there (regardless an issue should be filed).